app/code/Magento/Newsletter/Model/Subscriber.php

```diff
@@ -397,19 +432,16 @@ public function randomSequence($length = 32)
      * Subscribes by email
      *
      * @param string $email
+     * @param int $storeId
      * @throws \Exception
      * @return int
      *
      * @SuppressWarnings(PHPMD.CyclomaticComplexity)
      * @SuppressWarnings(PHPMD.NPathComplexity)
      */
-    public function subscribe($email)
+    public function subscribe($email, $storeId)
```

Comment:
Unfortunately this class marked as @api, so according to https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/ we can't add new parameters to existing public methods